addons/account/static/tests/tours/account_product_catalog_tests.js

```diff
@@ -0,0 +1,24 @@
+import { registry } from "@web/core/registry";
+
+registry.category("web_tour.tours").add('test_use_product_catalog_on_invoice', {
+    steps: () => [
+    {
+        content: "Click Catalog Button",
+        trigger: 'button[name=action_add_from_catalog]',
+        run: "click",
+    },
+    {
+        content: "Add a Product",
+        trigger: '.o_product_kanban_catalog_view .o_kanban_record:has(span:contains("Large Cabinet")) .o_product_catalog_buttons .fa-shopping-cart',
+        run: "click",
+    },
+    {
+        content: "Back to Invoice",
+        trigger: '.o-kanban-button-back',
+        run: "click",
+    },
+    {
+        content: "Check valid Product",
+        trigger: '.o_field_product_label_section_and_note_cell span:contains("Large Cabinet")',
+    },
+]});
```

Comment:
As you will need to force-push, can you also make sure the code style is ok ?
https://github.com/odoo/odoo/wiki/Javascript-coding-guidelines#tooling